### ISSUE TITLE
Compiling v8.1 on Mac OS 10.11 is failing with Node 6 and NPM 3

### ISSUE THREAD

**Issue by cn-yn** [KeitIG/museeks#342]
Same issue that hipunk was having in Gitter -- saw that changing Node versions seemed to resolve this for them, but I was already using the LTS version and no matter what wrangling I do I get the same compilation error where Babel doesn't seem to be transpiling anything. Moved down to 8.0 and it compiles fine -- prebuilt version of 8.1 rocking for now though so I'm getting to check it out, but interested to tinker under the hood a bit.

**Comment by KeitIG** [comment]
@cn-yn Hello ! Are you trying to compile `master` or the 0.8.1 tag ?

**Comment by KeitIG** [comment]
(I'm currently using node 6.10.3, and npm 3.10.10)

**Comment by cn-yn** [comment]
I tried both the master and the tag yesterday -- node 6.10.3 and npm 3.10.10 here as well. 

Tried building master just now in my ~/Downloads folder rather in the global Electron path (contents/resources etc etc) and it went fine. I'm guessing some babelrc or npm setting are getting borked by the electron package when you try to do that on Mac. Either way just building it wherever then running electron path-to-museeks is working great here.

Thanks for the help, sorry for the false alarm! Looking forward to some customization/tinkering - thanks!
